pod-security-cel/baseline/disallow-capabilities/disallow-capabilities.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: disallow-capabilities
+  annotations:
+    policies.kyverno.io/title: Disallow Capabilities in CEL expressions
+    policies.kyverno.io/category: Pod Security Standards (Baseline) in CEL
+    policies.kyverno.io/severity: medium
+    policies.kyverno.io/minversion: 1.11.0
+    kyverno/kubernetesVersion: "1.22-1.23"
```

Comment:
This annotation key isn't right.

Reply:
It's right for Artifact Hub but internally we use a different form.

Reply:
I fixed it. Thank you.